run-benchmark.sh

```diff
@@ -1011,18 +1046,12 @@ runQueries() {
 	  # TODO: implement TPCH validation checks
 
 	  # now start running
-	  if [[ $ITER -le 1 ]]; then
-	    echo "Handle single run"
-	    runQueries $SCHEMA
-
-	    # output single results
-	    checkBenchResults $SCHEMA 0
+	  BENCH_PASS="PASS"
+      > ${LOGBASE}logs/test_run.csv
```

Comment:
Use LOGDIR

Reply:
Jenkins doesn't know about LOGDIR. run.csv and test_run.csv are consumed by Jenkins. Potentially, there may many LOGDIR's - which one to use?